### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by tenisakb**

### CODE REVIEW

File: src/components/shared/loading-spinner.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    <Spinner>
+      <div />
+      <div />
+      <div />
+      <div />
+    </Spinner>
+  </ThemeProvider>
+);
+
+export default LoadingSpinner;
```
Comment (EdmundMai) [inline src/components/shared/loading-spinner.tsx]: can we just use a library for this? or is it so customized that we need to do this?

https://github.com/davidhu2000/react-spinners
https://github.com/dmitrymorozoff/react-spinners-kit
https://github.com/tsuyoshiwada/react-md-spinner

File: src/components/stands-out/stands-out-slide.tsx [hunk 2]
```diff
@@ -669,11 +669,11 @@ const SubscriptionSlide = ({ history, isLoggedIn }) => (
...
-          <CommunityButton onClick={() => history.push("/login")}>
-            Login
-          </CommunityButton>
-          <CommunityButtonSecondary onClick={() => history.push("/sign-up")}>
+          <CommunityButton onClick={() => history.push("/sign-up")}>
             Sign up
+          </CommunityButton>
+          <CommunityButtonSecondary onClick={() => history.push("/login")}>
+            Login
           </CommunityButtonSecondary>
```
Comment (EdmundMai) [inline src/components/stands-out/stands-out-slide.tsx]: make these `a`, not buttons

File: src/components/shared/loading-spinner.tsx (hunk 1 above)
Comment (tenisakb) [reply src/components/shared/loading-spinner.tsx]: fixed

File: src/components/stands-out/stands-out-slide.tsx (hunk 2 above)
Comment (tenisakb) [reply src/components/stands-out/stands-out-slide.tsx]: fixed

File: src/components/signup/send-payment-slide.tsx [hunk 3]
```diff
@@ -175,6 +180,9 @@ class SendPaymentSlide extends Component<
             complete setup of your account once the network transaction is
             confirmed. Please be patient.
           </Content>
+          <Wrapper>
+            <Spinner isActive={true} className={""} />
```
Comment (EdmundMai) [inline src/components/signup/send-payment-slide.tsx]: remove classname

File: src/components/shared/spinner.tsx [hunk 4]
```diff
@@ -2,7 +2,15 @@ import React from "react";
...
 
 const Spinner = ({ isActive, className }) => {
-  return isActive ? <MDSpinner className={className} /> : null;
+  return isActive ? (
+    <MDSpinner
+      color1={"#2e6dde"}
+      color2={"#2e6dde"}
+      color3={"#2e6dde"}
+      color4={"#2e6dde"}
+      className={className}
```
Comment (EdmundMai) [inline src/components/shared/spinner.tsx]: don't need classname

File: src/components/stands-out/stands-out-slide.tsx [hunk 5]
```diff
@@ -668,14 +681,14 @@ const SubscriptionSlide = ({ history, isLoggedIn }) => (
             </CommunityColumn>
           </CommunityWrapper>
         </CommunityContainer>
-        <CommunityButtonWrapper>
-          <CommunityButton onClick={() => history.push("/login")}>
-            Login
-          </CommunityButton>
-          <CommunityButtonSecondary onClick={() => history.push("/sign-up")}>
+        <CommunityLinkWrapper>
+          <CommunityLink onClick={() => history.push("/sign-up")}>
```
Comment (EdmundMai) [inline src/components/stands-out/stands-out-slide.tsx]: use href, not onClick. it's an anchor tag

File: src/components/stands-out/stands-out-slide.tsx [hunk 6]
```diff
@@ -668,14 +681,14 @@ const SubscriptionSlide = ({ history, isLoggedIn }) => (
...
             Sign up
-          </CommunityButtonSecondary>
-        </CommunityButtonWrapper>
+          </CommunityLink>
+          <CommunityLinkSecondary onClick={() => history.push("/login")}>
```
Comment (EdmundMai) [inline src/components/stands-out/stands-out-slide.tsx]: use href, not onClick. it's an anchor tag

File: src/components/stands-out/stands-out-slide.tsx (hunk 5 above)
Comment (EdmundMai) [reply src/components/stands-out/stands-out-slide.tsx]: actually I think you should use 
`import { Link } from "react-router-dom";`

 look at src/components/shared/header.tsx. it allows us to link using react-router so we don't need to refresh the page.

ex: `<Link to="/sign-up">Sign up</Link>` you can style it too using styled components

File: src/components/stands-out/stands-out-slide.tsx (hunk 6 above)
Comment (tenisakb) [reply src/components/stands-out/stands-out-slide.tsx]: fixed

File: src/components/stands-out/stands-out-slide.tsx (hunk 5 above)
Comment (tenisakb) [reply src/components/stands-out/stands-out-slide.tsx]: fixed

File: src/components/shared/spinner.tsx (hunk 4 above)
Comment (tenisakb) [reply src/components/shared/spinner.tsx]: fixed

File: src/components/signup/send-payment-slide.tsx (hunk 3 above)
Comment (tenisakb) [reply src/components/signup/send-payment-slide.tsx]: fixed

**Pull request merged**